lib/RequestManager.ts

```diff
@@ -19,9 +19,10 @@ export interface MakeRequestOptions {
 }
 
 const clientVersion = require('../package.json').version;
+const isNodeEnv = (typeof process !== 'undefined') && (get(process, 'release.name') === 'node');
```

Comment:
obnoxious: `process.env` is available in our webpack builds.  

we'll probably want to revisit this a bit further, as the `https/Agent` import also causes Webpack to add a whole bunch of shims that we don't need.